CONTRIBUTING.md

```diff
@@ -6,5 +6,15 @@ Please be aware, BeeWare operates under a Code of Conduct.
 
 See [CONTRIBUTING to BeeWare](http://beeware.org/contributing) for details.
 
-See also the Briefcase specific contributing guides for contributing [code](https://briefcase.readthedocs.io/en/latest/how-to/contribute-code.html) and [documentation](https://briefcase.readthedocs.io/en/latest/how-to/contribute-docs.html)
+See also the Briefcase specific contributing guides for contributing
+[code](https://briefcase.readthedocs.io/en/latest/how-to/contribute-code.html)
+and
+[documentation](https://briefcase.readthedocs.io/en/latest/how-to/contribute-docs.html)
 
+## Development Environment
```

Comment:
This note should be added to the contribution guide that is linked on L10, rather than here.

If I'm reading the pre-commit docs right, we also need to instruct users to run `pre-commit install`; just installing pre-commit won't install the hooks in a way that they'll actually run.

Reply:
We may also want to add some notes about what to expect if a local commit fails. AFAICT, the process is: 
* attempt a commit
* tests will run (show a console dump showing what to expect)
* Any files with formatting problems will have the black fixes applied, but unstaged
* Manually add those files, recommit.
* tests will run an pass (show a successful console dump)

Reply:
Done!